clang/test/Sema/warn-assignment-bool-context.c

```diff
@@ -0,0 +1,35 @@
+// RUN: %clang_cc1 -x c -fsyntax-only -Wparentheses -verify %s
+
+// NOTE: Don't know if tests allow includes.
+#include <stdbool.h>
```

Comment:
Include shouldn't be here, you can just define the thing manually if you'd like.